modServer.bas

```diff
@@ -2176,8 +2176,7 @@ Sub ShutdownServer()
     MagicRS.Close
     DB.Close
     WS.Close
-    
-    End

```

Comment:
Was this `End` just an error?

Reply:
Yes, when you unload all forms it ends the program without even needing to use End